### PR TITLE
Accessibility issue reports

### PR DESCRIPTION
**Pull request by tanveerisonline**
Page titles - "no issues found".
Heading Tag - "no issues found".
Interaction - no issues found" 
Color contrast -"no issues found".  (Already taken from Figma design).
Alt Attribute Added to the mobile menu image.
Resize - Font sizes increased for good readability.
Moving Content - "There is no moving content on this website".
Multimedia - "No audio and Video used for this website".
The Basic Structure - "No issue found".

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <header class="header" id="fixed">
       <nav>
         <a href="#" target="_parent" class="logo">My Portfolio</a>
-        <img src="./assets/Union.png" class="mobile_menu_icon"/>
+        <img src="./assets/Union.png" class="mobile_menu_icon" alt="mobile-menu-icon"/>
```
Comment (tanveerisonline) [inline index.html:17]: Added alt attribute here

File: queries.css [hunk 2]
```diff
@@ -154,6 +154,7 @@
     margin: 10px 0;
     padding: 10px 0;
     color: #344563;
+    font-size: 20px;
```
Comment (tanveerisonline) [inline queries.css:157]: Increased font size in desktop version cards for better readability.

File: queries.css [hunk 3]
```diff
@@ -232,6 +233,10 @@
     margin: 50px 0 0 80px;
   }
 
+  .about .about_details {
+    font-size: 20px;
```
Comment (tanveerisonline) [inline queries.css:237]: Increased font size in the About section of the desktop version for better readability.

File: styles.css [hunk 4]
```diff
@@ -155,6 +155,7 @@ nav {
   margin: 10px 0;
   padding: 10px 0;
   color: #344563;
+  font-size: 18px;
```
Comment (tanveerisonline) [inline styles.css:158]: Increased font size in cards for better readability in the mobile version.

File: styles.css [hunk 5]
```diff
@@ -206,6 +207,7 @@ nav {
 
 .about .about_details {
   color: #344563;
+  font-size: 18px;
```
Comment (tanveerisonline) [inline styles.css:210]: Increased font size in the about section for better readability in the mobile version.

**Review by tanveerisonline — COMMENTED** [review]
### Accessibility Report :grinning:

1. Page titles - "no issues found".
2. Heading Tag - "no issues found".
3. Interaction - no issues found"
4. Color contrast -"no issues found". (Already taken from Figma design).
5. Alt Attribute Added to the mobile menu image.
6. Resize - Font sizes increased for good readability.
7. Moving Content - "There is no moving content on this website".
8. Multimedia - "No audio and Video used for this website".
9. The Basic Structure - "No issue found".

File: index.html (hunk 1 above)
Comment (asifaminisonline) [inline index.html:17]: Added alt attribute here

File: queries.css (hunk 2 above)
Comment (asifaminisonline) [inline queries.css:157]: Increased font size in desktop version cards for better readability.

File: queries.css (hunk 3 above)
Comment (asifaminisonline) [inline queries.css:237]: Increased font size in the About section of the desktop version for better readability.

**Review by asifaminisonline — COMMENTED** [review]
**### Accessibility Report 😀**

1. Alt Attribute Added to the mobile menu image.
2. Resize - Font sizes increased for good readability.

**Review by andgarzonmal — APPROVED** [review]
Hi @tanveerisonline, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**